lib/rebaser-server/src/config.rs

```diff
@@ -168,10 +156,6 @@ fn default_cyclone_encryption_key_path() -> String {
     "/run/rebaser/cyclone_encryption.key".to_string()
 }
 
-fn default_recreate_management_stream() -> bool {
-    false
-}
```

Comment:
This was never used and we'll want to think about configuration differently in the long term anyway with N rebaser servers.